src/components/molecules/Navbar/index.jsx

```diff
@@ -59,145 +59,137 @@ class Navbar extends React.Component {
-                          pathname: "/",
-                          state: {
-                            actionCard: 1,
-                          },
-                        }}
-                      >
-                        <MDBBtn color="green" size="md">
-                          Sign In
-                        </MDBBtn>
-                      </Link>
-                    )}
-                  </>
-                )}
-              </MDBNavbarNav>
-            </MDBCollapse>
-          </MDBContainer>
-        </MDBNavbar>
-        {this.state.showSettings && (
-          <Suspense fallback={<div>Loading...</div>}>
-            <Settings
-              {...this.props}
-              closeModal={this.handleSettingsClose}
-              saveSettings={globalFunctions.saveSettings}
-            />
-          </Suspense>
-        )}
-      </>
+      <MDBNavbar color="light" light expand="md">
+        <MDBContainer>
+          {location.pathname === "/" ? (
+            <MDBSmoothScroll to="home" className="d-inline">
+              <MDBNavbarBrand className="flex-center">
+                <img
+                  src={SNEKLogo}
+                  alt="SNEK Logo"
+                  className="img-fluid mr-2"
+                />
+                <span className="font-weight-bold">SNEK</span>
+              </MDBNavbarBrand>
+            </MDBSmoothScroll>
+          ) : (
+            <>
+              {!loggedUser.anonymous ? (
+                <Link to={"/u/" + loggedUser?.username}>
+                  <MDBNavbarBrand className="flex-center">
+                    <img
+                      src={SNEKLogo}
+                      alt="SNEK Logo"
+                      className="img-fluid mr-2"
+                    />
+                    <span className="font-weight-bold">SNEK</span>
+                  </MDBNavbarBrand>
+                </Link>
+              ) : (
+                <Link to="/">
+                  <MDBNavbarBrand className="flex-center">
+                    <img
+                      src={SNEKLogo}
+                      alt="SNEK Logo"
+                      className="img-fluid mr-2"
+                    />
+                    <span className="font-weight-bold">SNEK</span>
+                  </MDBNavbarBrand>
+                </Link>
+              )}
+            </>
+          )}
+          <MDBNavbarToggler onClick={this.toggleCollapse} />
+          <MDBCollapse id="navbarCollapse" isOpen={this.state.isOpen} navbar>
+            <MDBNavbarNav left>
+              <MDBNavItem>
+                <SearchBar />
+              </MDBNavItem>
+            </MDBNavbarNav>
+            <MDBNavbarNav right>
+              {!loggedUser.anonymous ? (
+                <>
+                  <div className="spacer" />
+                  <MDBNavItem>
+                    <MDBDropdown>
+                      <MDBDropdownToggle nav caret>
+                        <img
+                          src={loggedUser.avatarUrl}
```

Comment:
This doesn't seem to work yat.
![image](https://user-images.githubusercontent.com/26285351/86581902-0db6ed80-bf81-11ea-957a-32c782bb454b.png)

Reply:
This will work after the backend update, currently the logged in user only has a valid username. The profile picture is set static.